tools/hooks/requirements_changed.rb

```diff
@@ -7,7 +7,8 @@
   "package.json" => "yarn",
   "bower.json" => "bower install",
   "Gemfile" => "bundle install",
-  "Berksfile" => "berks install"
+  "Berksfile" => "berks install",
+  "schema.rb" => "rake db:migrate"
```

Comment:
Currently these "file patterns" do not accept paths. They do an exact comparison to the result of [File.basename](https://ruby-doc.org/core-1.9.3/File.html#method-c-basename). I don't want to risk breaking the rest of what's already working in this file so I think we should punt on that until we need it.